src/kernel/src/main.rs

```diff
@@ -594,10 +594,6 @@ struct Args {
     #[arg(long)]
     #[serde(default)]
     pro: bool,
-
-    #[arg(long)]
-    #[serde(default)]
-    idps: ConsoleId,
```

Comment:
What wrong with this?

Reply:
Nothing, I just forgot to exclude it from the commit. I just removed it so that I can actually run the emualator, which is not possible right now, because the argument is required

Reply:
Actually... now that I run the emulator again, it's fine. But I swear that previously it would complain about the argument not being specified :eyes: .